README.md

```diff
@@ -90,6 +90,7 @@ container.
         // Global variables passed to twig templates
         'ga_tracking' => 'UA-XXXXX-X'
     ],
+    'timezone' => 'default timezone identifier, e.g.: America/New_York'
```

Comment:
I wasn't sure which convention to adopt here, since the ga_tracking example is demonstrative while the lines above are descriptive.